examples/with-shell-commands/apps/apps-a/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "app-a",
+  "description": "A workspace to use as a model for a final application that the rest of your dependencies are building towards.",
+  "scripts": {
+    "build": "sleep 4 && mkdir -p dist && echo \"Your application output!\" > dist/app-output.txt && echo \"Application A is built!\"",
+    "lint": "sleep 4 && echo \"Linted!\"",
+    "type-check": "sleep 4 && echo \"Types checked!\""
+  },
+  "dependencies": {
+    "pkg-a": "workspace:*",
+    "with-workspace-configuration": "workspace:*"
+  },
+  "devDependencies": {
```

Comment:
Is the distinction between `dependencies` and `devDependencies` even useful? I'm leaning towards no

Reply:
I'll say yeah because it's meant to mimic a tooling configuration which would likely be a devDep. (`eslint`, `typescript`, etc.)

Reply:
Every example probably doesn't need to show every feature, fwiw. This example is about removing frameworks, since turbo itself doesn't treat dep vs devDep differently, I don't see the value in adding it to this example. In fact, it's _reintroducing_ ideas about specific tools into the example (the very thing we're trying to remove!)

Reply:
Fair play, took it out.